command/plugin_register.go

```diff
@@ -81,22 +81,27 @@ func (c *PluginRegisterCommand) Flags() *FlagSets {
 		Name:       "command",
 		Target:     &c.flagCommand,
 		Completion: complete.PredictAnything,
-		Usage: "Command to spawn the plugin. This defaults to the name of the " +
-			"plugin if both oci_image and command are unspecified.",
+		Usage: "Command to spawn the plugin. If sha256 is provided to register with a plugin binary, " +
```

Comment:
Suggest to refer to sha256 as a CLI option.
```suggestion
		Usage: "Command to spawn the plugin. If -sha256 is provided to register with a plugin binary, " +
```